src/prefect/settings/models/server/database.py

```diff
@@ -148,14 +148,23 @@ class ServerDatabaseSettings(PrefectBaseSettings):
 
     sqlalchemy_pool_size: Optional[int] = Field(
         default=None,
-        description="Controls connection pool size when using a PostgreSQL database with the Prefect API. If not set, the default SQLAlchemy pool size will be used.",
+        description="Controls connection pool size of database connection pools from the Prefect API. If not set, the default SQLAlchemy pool size will be used.",
         validation_alias=AliasChoices(
             AliasPath("sqlalchemy_pool_size"),
             "prefect_server_database_sqlalchemy_pool_size",
             "prefect_sqlalchemy_pool_size",
         ),
     )
 
+    connection_app_name: Optional[str] = Field(
+        default=None,
+        description="Controls the application_name field for connections opened from the connection pool when using a PostgreSQL database with the Prefect API.",
+        validation_alias=AliasChoices(
+            AliasPath("connection_app_name"),
+            "prefect_server_database_connection_app_name",
+        ),
```

Comment:
These validation aliases should only be needed for backwards compatibility, so I don't think they need to be added in this case.
```suggestion
```

Reply:
ah that makes sense - will remove